app/graphql/queries/week_in_reviews/query_manifest.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Queries
+  module WeekInReviews
+    # Module that:
```

Comment:
Is this - or should this be - YARN format?

Reply:
Yeah, this is the yarn format.  All good.